packages/react-cosmos-webpack/src/webpack-config.js

```diff
@@ -44,11 +44,13 @@ export default function getWebpackConfig(
     publicPath: '/loader/',
   };
 
-  const loaders = userWebpackConfig.module && userWebpackConfig.module.loaders ?
-    [...userWebpackConfig.module.loaders] : [];
+  // use what user wants
+  const webpackRulesOptionName = userWebpackConfig.module && userWebpackConfig.module.rules ? 'rules' : 'loaders';
```

Comment:
I guess they wanted old configs to still work while providing an improved loader API for those who have the time to upgrade.